frameworks/cassandra/src/main/dist/cassandra.yaml

```diff
@@ -0,0 +1,100 @@
+cluster_name: {{CASSANDRA_CLUSTER_NAME}}
+rpc_address: {{LIBPROCESS_IP}}
+listen_address: "{{TASK_NAME}}.{{FRAMEWORK_NAME}}.mesos"
+hints_directory: {{MESOS_SANDBOX}}/container-path/hints
+data_file_directories:
+  - {{MESOS_SANDBOX}}/container-path/data
+commitlog_directory: {{MESOS_SANDBOX}}/container-path/commitlog
+saved_caches_directory: {{MESOS_SANDBOX}}/container-path/saved_caches
+seed_provider:
+    - class_name: org.apache.cassandra.locator.SimpleSeedProvider
+      parameters:
+          - seeds: "node-0-server.{{FRAMEWORK_NAME}}.mesos,node-1-server.{{FRAMEWORK_NAME}}.mesos"
```

Comment:
We need to support seed discovery from a different datacenter and periodically synchronize them.